tool/tctl/common/device/resource.go

```diff
@@ -34,22 +34,16 @@ type Resource struct {
 }
 
 // ResourceSpec is the device resource specification.
+// This spec is intended to closely mirror `devicepb.Device` but swaps some data around
```

Comment:
Use godoc link?

```suggestion
// This spec is intended to closely mirror [devicepb.Device] but swaps some data around
```

https://go.dev/doc/comment#doclinks